### ISSUE TITLE
Unable to change Background when configuring page template in Wayland

### ISSUE THREAD

**Issue by f1lipe** [xournalpp/xournalpp#3228]
**Affects versions : Linux
 - (Linux only) Desktop environment: Gnome-Wayland
 - Which version of libgtk do you use : 3.24.30
 - Version of Xournal++: 1.0.20-hotfix (git commit adaa97ff)
 - Installation method: distribution package manager

**Describe the bug**
Unable to change Background when configuring page template in Wayland

**To Reproduce**
1. Click on menu Journal > Configure Page Template
2. Click on arrow button to select the background

**Expected behavior**
When step 2 is executed, the expected behavior is to popup a small dialog with the available templates. But, instead, nothing happens.

**Additional context**
it works fine when using a Xorg session.

**Comment by rolandlo** [comment]
Does that happen in [version 1.1.0+dev](https://github.com/xournalpp/xournalpp/releases/tag/nightly) for you as well? I can't reproduce it, even when on Gnome-Wayland.

**Comment by f1lipe** [comment]
> Does that happen in [version 1.1.0+dev](https://github.com/xournalpp/xournalpp/releases/tag/nightly) for you as well? I can't reproduce it, even when on Gnome-Wayland.

Yes, I just updated to Xournal++ 1.1.0 (commit 9d1277de) and the issue is still there. 
Let me know if there is any additional test that I can execute.

**Comment by rolandlo** [comment]
You could start `xournalpp` from terminal and see if there are any error messages. 
Two more questions: 
1. Do you use a second monitor?
2. Do you use a modified `pagetemplates.ini`
